src/components/atoms/OcTag/OcTag.vue

```diff
@@ -45,6 +45,12 @@ export default {
         return value.match(/(small|medium|large)/)
       },
     },
+
+    rounded: {
```

Comment:
Could you add a description here to keep is consistent with the other props?

Reply:
sure :)